pyproject.toml

```diff
@@ -111,11 +111,6 @@ notes = """,
     XXX,
     """
 
-[tool.pylint-nautobot]
-supported_nautobot_versions = [
-    "2"
-]
-
 [tool.ruff]
 line-length = 120
 target-version = "py39"
```

Comment:
does this need to change to py310?

Reply:
It looks like that was missed in the `cookiecutter-nautobot-app` repo. I will open a backlog item and get that fixed.